contribute/release-process.md

```diff
@@ -425,6 +425,10 @@ The wheel files will be generated at each platform directory under `pulsar-clien
 
 This step is for every release. Read the specific guide for [writing release notes](release-note-guide.md).
 
+### Older download
+
+To display the historical version download content on [this page](https://pulsar.apache.org/download/#older-releases), you need to add versions to the [release.json](https://github.com/apache/pulsar-site/blob/main/releases.json).
+
```

Comment:
Shouldn't it already be covered here: https://pulsar.apache.org/contribute/release-process/#update-the-site?

> For every release, you should add a <release-version> entry to the corresponding place in the releases.json file.